src/components/DateTimePicker/Internal/ocpicker.module.scss

```diff
@@ -451,29 +577,78 @@
   }
 
   &-footer {
-    border-top: $picker-border;
+    font-family: var(--font-stack-full);
+    padding: 0 $space-m;
+  }
+
+  &-footer-border {
+    border-top: var(--picker-border-width) var(--picker-border-style)
+      var(--grey-background2-color);
     display: flex;
-    font-family: $octuple-font-family;
+    height: 100%;
```

Comment:
this makes it 100% of footer or what? Seems a bit odd

Reply:
Yes, this is to account for the border padding. It's weird, but works a treat.